src/codegen.cpp

```diff
@@ -6432,6 +6432,7 @@ extern "C" void jl_init_codegen(void)
 #endif
 
 #ifdef USE_POLLY
+    cl::ParseEnvironmentOptions("Julia", "JULIA_POLLY_ARGS");
```

Comment:
Do you need a separate environment variable? You could also enable `JULIA_LLVM_ARGS`[1] when polly is used and then we have only one way of passing environment variables to llvm.

[1]https://github.com/JKrehl/julia/blob/faee750b2e7f987830faed16b7982707fcf6973b/src/codegen.cpp#L6421